### ISSUE TITLE
[Feature Request] Warn on missing Title Update/DLC

### ISSUE THREAD

**Issue by ekuland** [Ryujinx/Ryujinx#4304]
#### Overview:
- Warn users when applied updates/DLCs are not located

#### Smaller Details: 
- When attempting to launch a title and any files are missing from the currently selected updates/dlc, do not launch and display a warning dialog with the details as to which paths are missing.

#### Nature of Request:  
- Addition

### Why would this feature be useful?
It is quite easy when moving or renaming files to forget to reapply updates or DLCs before the next launch. This can lead to mod/cheat crashes or save corruption in many cases.

**Comment by tvoracek** [comment]
Aren't updates applied to base game and then it is possible to delete the update files (as Yuzu does it, not sure about Ryujinx)? If yes then this request doesn't make much sense.

**Comment by MutantAura** [comment]
If you manually move an update file and forget to update its new directory in Ryujinx, it will simply boot 1.0.0 or whatever is on the dump. The request is asking that Ryujinx flags when it has to skip applying the update due to a directory error instead of just catching it with a skip.

**Comment by tvoracek** [comment]
Can you assign this to me?

**Comment by gnisman** [comment]
@tvoracek are you working on it? If not i could do it

**Comment by tvoracek** [comment]
@gnisman Yes I started some work, not finished yet. Will need to update my code with latest master and then continue working on it.
